### DISCUSSION TITLE
default browser directory

### DISCUSSION THREAD

**Discussion by rawr1de** [cmus/cmus#1218] [Troubleshooting] [answered]
I've searched around and spent some time reading the docs but couldn't find anything related to, how to change the default browser directory, when hitting 5 it shows my home directory, I would like to point out my music dir as the default browser dir. I use mostly views 1 and 5.

**Comment by majava3000** [comment, marked as answer]
You might achieve what you want by switching the working directory on cmus startup. The working directory will be used as the default starting location for the browser.

So, something like this in your rc file (mine is ~/.config/cmus/rc, adjust with your correct path as required):
`cd /auto/audio/library`

**Reply by rawr1de** [reply]
Couldn't get it working.
I use the same path for my config file (~/.config/cmus/rc) 
Just threw `cd /auto/audio/library` at my rc file but it did nothing.
I'm wondering, where did you found this info? There's nothing on man pages stating default audio library locations.

**Reply by majava3000** [reply]
1. replace `/auto/audio/library` with the path that you have your music in (/auto/audio/library was just an example)
2. restart cmus, so that it will read and apply rc
3. browser view should be by default in the directory

`cd` is documented in the cmus manual page (raw version of which you can find here https://github.com/cmus/cmus/blob/master/Doc/cmus.txt#L427). Browser seems to start by default from the current working directory (I had a hunch about this). I tested it out and it worked for me.
